docs/user-guide/docs-source/aws-setup.md

```diff
@@ -4,14 +4,14 @@ Ambit integrates with [Amazon Simple Storage Service (Amazon S3)](https://docs.a
 
 ## Setup
 
-To successfully utilize the S3 bucket, AWS credentials need to be configured in your environment for a role that has access to the S3 bucket. Follow [this guide](https://docs.aws.amazon.com/sdk-for-cpp/v1/developer-guide/credentials.html) for step-by-step instructions.
+To successfully utilize an Amazon S3 bucket, AWS credentials need to be configured in your environment for a role that has access to the Amazon S3 bucket. Follow [this guide](https://docs.aws.amazon.com/sdk-for-cpp/v1/developer-guide/credentials.html) for step-by-step instructions.
```

Comment:
The service name of Simple Storage Service should be abbreviated as "Amazon S3." The resource of the "buckets" in that service is called "S3 bucket." 
No need to edit instances of "S3 bucket," (unless they just say "bucket") but any instance of "S3" that is referring to the service should be "Amazon S3."

Reply:
I'll fix this in the next PR which is focused in part on consistency.